docs/security/trust/trust_sandbox.md

```diff
@@ -274,49 +241,59 @@ What happens when data is corrupted and you try to pull it when trust is
 enabled? In this section, you go into the `sandboxregistry` and tamper with some
 data. Then, you try and pull it.
 
-1. Leave the sandbox container running.
+1. Leave the `trustsandbox` shell and container running.
+
+2. Open a new interactive terminal from your host into the `sandboxregistry`.
 
-2. Open a new bash terminal from your host into the `sandboxregistry`.
+        $ docker exec -it sandboxregistry sh
+        #
 
-        $ docker exec -it sandboxregistry bash
-        296db6068327#
+3. List the layers for the `test/trusttest` image you pushed:
 
-3. Change into the registry storage.
+        # ls -l /var/lib/registry/docker/registry/v2/repositories/test/trusttest/_layers/sha256
+        total 12
+        drwxr-xr-x 2 root root 4096 Jun 10 17:26 a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4
+        drwxr-xr-x 2 root root 4096 Jun 10 17:26 aac0c133338db2b18ff054943cee3267fe50c75cdee969aed88b1992539ed042
+        drwxr-xr-x 2 root root 4096 Jun 10 17:26 cc7629d1331a7362b5e5126beb5bf15ca0bf67eb41eab994c719a45de53255cd
 
-    You'll need to provide the `sha` you received when you pushed the image.
+4. Change into the registry storage for one of those layers.
 
         # cd /var/lib/registry/docker/registry/v2/blobs/sha256/aa/aac0c133338db2b18ff054943cee3267fe50c75cdee969aed88b1992539ed042
 
-4. Add malicious data to one of the trusttest layers:
+5. Add malicious data to one of the trusttest layers:
 
         # echo "Malicious data" > data
 
-5. Got back to your sandbox terminal.
+6. Go back to your `trustsandbox` terminal.
 
-6. List the trusttest image.
+7. List the trusttest image.
 
-        # docker images | grep trusttest
-        docker/trusttest                 latest              a9539b34a6ab        7 weeks ago         5.025 MB
-        sandboxregistry:5000/test/trusttest   latest              a9539b34a6ab        7 weeks ago         5.025 MB
-        sandboxregistry:5000/test/trusttest   <none>              a9539b34a6ab        7 weeks ago         5.025 MB
+        / # docker images | grep trusttest
+        REPOSITORY                            TAG                 IMAGE ID            CREATED             SIZE
+        docker/trusttest                      latest              cc7629d1331a        11 months ago       5.025 MB
+        sandboxregistry:5000/test/trusttest   latest              cc7629d1331a        11 months ago       5.025 MB
+        sandboxregistry:5000/test/trusttest   <none>              cc7629d1331a        11 months ago       5.025 MB
 
-7. Remove the `trusttest:latest` image.
+8. Remove the `trusttest:latest` image.
```

Comment:
It might be worth clarifying that we're doing this to clear our local cache, to test that we won't download malicious data from the registry on `pull`?